src/index.coffee

```diff
@@ -9,24 +9,31 @@ class RollbarStream extends stream.Writable
     streamWritableOpts.highwaterMark ?= 100
     super(streamWritableOpts)
 
-    opts.branch ?= 'master'
-    opts.root ?= process.cwd()
-    opts.host ?= process.env.DYNO
-
     @client = opts.client ? do ->
-      rollbar.init opts.token, _(opts).omit('token')
-      rollbar
+      config = _(opts)
+        .chain()
+        .omit('token')
+        .defaults({
+          accessToken: opts.token
+          environment: process.env.NODE_ENV
```

Comment:
we weren't setting environment before? shouldn't this be `APP_INSTANCE`? we don't have staging and production to be conflated (although we stopped using Rollbar on staging for servers a while ago... we might re-enable at some point)

Reply:
I opted to remove any handling of `environment` here since we didn't do it before.  The docs for `goodeggs-logger` suggest setting `environment` explicitly, and we seem pretty good about sourcing that from `APP_INSTANCE`, eg:

```
logger.addRollbar({token: process.env.ROLLBAR_TOKEN, environment: process.env.APP_INSTANCE});
```